src/scripts/zero_patch_fs.lua

```diff
@@ -0,0 +1,85 @@
+
+local file_system = {}
```

Comment:
I think it's probably be better to read in the 'fs.lua' file for each call to zero_patch_fs.getFiles().
It should result in less persistent memory being used at any given time.